boards/pico_explorer_base/src/main.rs

```diff
@@ -414,8 +414,9 @@ pub unsafe fn start() -> (
             20 => peripherals.pins.get_pin(RPGpio::GPIO20),
             21 => peripherals.pins.get_pin(RPGpio::GPIO21),
             22 => peripherals.pins.get_pin(RPGpio::GPIO22),
-            23 => peripherals.pins.get_pin(RPGpio::GPIO23),
-            24 => peripherals.pins.get_pin(RPGpio::GPIO24),
+            26 => peripherals.pins.get_pin(RPGpio::GPIO26),
+            27 => peripherals.pins.get_pin(RPGpio::GPIO27),
+            28 => peripherals.pins.get_pin(RPGpio::GPIO28),
```

Comment:
Looking at the schematic and the picture posted in the PR, it looks like only GP0-GP7 are exposed as general purpose pins, with GP0 & GP1 actually serving as TX/RX for UART;

GP20 GP21 GP22 are I2C, and GP26-28 are (as @bradjc points out) ADC channels 0, 1, & 2

Reply:
So then, should GP20-GP22 also be deleted to only expose GP0-GP7?